dev-libs/libsigsegv/libsigsegv-2.11.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="A library for handling page faults in user mode"
+DESCRIPTION="GNU libsigsegv is a library for handling page faults in user mode. \
+ A page fault occurs when a program tries to access to a region of memory that \
+ is currently not available. Catching and handling a page fault is a useful \
+ technique for implementing:
+ * Pageable virtual memory
+ * Memory-mapped access to persistent databases
+ * Generational garbage collectors
+ * Stack overflow handlers
+ * Distributed shared memory
+This library supports three sets of functions, all defined in <sigsegv.h>:
+ * Global SIGSEGV handlers
+ * Local SIGSEGV handlers (a handler per memory area)
+ * Stack overflow handlers"
+HOMEPAGE="https://www.gnu.org/software/libsigsegv"
+COPYRIGHT="1998-2011 Bruno Haible
+	2002-2009 Paolo Bonzini
+	2008-2010 Eric Blake"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://ftp.gnu.org/pub/gnu/libsigsegv/libsigsegv-$portVersion.tar.gz"
+CHECKSUM_SHA256="dd7c2eb2ef6c47189406d562c1dc0f96f2fc808036834d596075d58377e37a18"
+SOURCE_DIR="libsigsegv-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libsigsegv$secondaryArchSuffix = $portVersion
+	lib:libsigsegv$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libsigsegv${secondaryArchSuffix}_devel = $portVersion
+	devel:libsigsegv$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libsigsegv$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize
+	cmd:autoconf
+	cmd:autoheader
+	cmd:automake
+	"
+
+BUILD()
+{
+	libtoolize --force --copy --install
+	aclocal -I m4
+	autoconf
+	autoheader
+	automake
+	runConfigure ./configure --prefix=$prefix --includedir=$includeDir \
```

Comment:
@waddlesplash Added mentioned changes to recipe, is this recipe ready to be merged?